ReSwiftExpectThunk.podspec

```diff
@@ -0,0 +1,29 @@
+Pod::Spec.new do |spec|
```

Comment:
I think this would probably be better suited as a subspec rather than a separate pod inside the same repo.
See https://guides.cocoapods.org/syntax/podspec.html#subspec and also https://guides.cocoapods.org/syntax/podspec.html#default_subspecs for omitting this from the default.

I would create two subspecs `Core`, and `Expect`, then set `Core` as the `default_subspec`

Reply:
I'm all for making it a subspec. I went with this route first to test it in a demo app. I will attempt a subspec next. Will report back on this.

Second thing I would be for adding a blurb in the README.md if that's cool.

Reply:
👍 👍

Reply:
I believe I got the subspec down. I have a test project here: https://github.com/jjgp/ExpectThunkDemo

I'd appreciate a sanity check since I did run into CocoaPods fun with a sigabrt at test time, however, I managed to resolve it.

Will work on the blurb in a bit.

Reply:
sigabrt will show up if the test target is a child target in the Podfile:

```
target 'ExpectThunkDemo' do
  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
  use_frameworks!

  # Pods for ExpectThunkDemo
  pod 'ReSwiftThunk', :git => 'git@github.com:jjgp/ReSwift-Thunk.git', :branch => 'expect-thunk-proposal'
  target 'ExpectThunkDemoTests' do
    inherit! :search_paths
    pod 'ReSwiftThunk/ExpectThunk', :git => 'git@github.com:jjgp/ReSwift-Thunk.git', :branch => 'expect-thunk-proposal'
  end
end
```